calculator.c

```diff
@@ -0,0 +1,109 @@
+#include<stdio.h>
+#include<string.h>
+#include <stdbool.h>
+bool error=false;//to check if nay error encountered
+bool isOperator(char character){//function to check for an operator
+    if(character=='+' || character=='*' || character=='/' || character=='-'){
+        return true;
+    }
+    return false;
+}
+int precedence(char operator){//to check precedence of the operator
```

Comment:
Method names should be like getPrecedence(). Please rename all methods accordingly.